proxy/logging/LogFormat.cc

```diff
@@ -550,6 +550,7 @@ LogFormat::parse_symbol_string(const char *symbol_string, LogFieldList *field_li
         Note("The log format symbol %s was not found in the "
```

Comment:
Where does this `Note` go? That's not sufficient?

Reply:
The problem is you see this once at startup.  (Or actually you see it 3 times for some bizarre reason.) On a live proxy, you get the other cryptic message a bazillion times, causing a fast diags.log roll over, so it's very unlikely you'll notice this message unless you know to look for it.  This is what happened to Dave Carlin.  Fortunately Leif had seen this in the past and remembered it, tipped me off.

Reply:
Yes, #4496 is trying to deal with the excessive fast rolling of `diags.log`. This kind of thing is why it needs to be fixed.